<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commands</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_open">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    padding-left: 21px; 
    padding-right: 21px;
    padding-top: 10.5px;   
    padding-bottom: 10.5px; 
	background-color: rgb(13, 110, 253);
	color: rgb(255, 255, 255);
}
QPushButton:disabled{
	background-color: rgb(43, 86, 180);
	color: rgb(178, 179, 180);
}
QPushButton:hover{
	background-color: rgb(11, 94, 215);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Open</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_sel_file">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding-left: 21px; 
    padding-right: 21px;
    padding-top: 10.5px;   
    padding-bottom: 10.5px;
	background-color: rgb(25, 135, 84);
	color: rgb(255, 255, 255);
 }
QPushButton:hover{
	background-color: rgb(21, 115, 71);
}</string>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sel_folder">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding-left: 21px; 
    padding-right: 21px;
    padding-top: 10.5px;   
    padding-bottom: 10.5px;
	background-color: rgb(25, 135, 84);
	color: rgb(255, 255, 255);
 }
QPushButton:hover{
	background-color: rgb(21, 115, 71);
}</string>
       </property>
       <property name="text">
        <string>Select Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_open</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_open</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
